src/sources/mysql/mysql-schema.lisp

```diff
@@ -69,8 +41,10 @@
   "Associate internal table type symbol with what's found in MySQL
   information_schema.tables.table_type column.")
 
-(defun filter-list-to-where-clause (filter-list &optional not)
+(defmethod filter-list-to-where-clause ((mysql copy-mysql) filter-list
+                                        &key not &allow-other-keys)
   "Given an INCLUDING or EXCLUDING clause, turn it into a MySQL WHERE clause."
+  (declare (ignore mysql))
```

Comment:
The declaration is not required in a method where you specialize on the argument in question. The compiler should not warn you about this argument not being used since you *already* use it for method dispatch.

Reply:
Fixed.